apps/editor/project.json

```diff
@@ -79,6 +89,15 @@
         "staticFilePath": "dist/apps/editor/browser",
         "spa": true
       }
+    },
+    "extract-enums": {
+      "executor": "nx:run-commands",
+      "configurations": {
+        "extract": {
+          "commands": ["ts-node libs/coral-island-enums/extractor.ts"]
+        }
+      },
+      "defaultConfiguration": "extract"
```

Comment:
I would have said to add an output to this task so that it's cached, but I remembered that I didn't enable caching 😂

Reply:
:D yeah i think optimization and stuff can be added later :)